railties/lib/rails/generators/app_base.rb

```diff
@@ -340,7 +340,15 @@ def spring_install?
       end
 
       def run_bundle
-        bundle_command('install') if bundle_install?
+        # Most of us have the CPUs; so let's parallelize `bundle install`
```

Comment:
@fxn Yes, the scheduler will treat threads (given they are user threads) the same as processes with respect to time on the CPU time (on Linux, at least). The scheduler doesn't really know the difference; it just considers them to be tasks. Therefore, they can bve scheduled in parallel. Now, implementations of Ruby will have different thread models, but theoretically threads can map to different cores.
 
@rafaelfranca re: the bundler configuration. I actually have this set for my personal setup; but how many newbies (or hell, even seasoned developers) know this exists? It's a rather low-level configuration thing and I would not expect a new user to have that configured. I think the *optimal* solution would be to have bundler parallel by default. As it stands now though, it is not and Rails users are suffering a slower `rails new ...`